gradio_tts_app.py

```
@@ -272,103 +299,134 @@ def load_and_prepare_model():
-        temperature=temperature,
-        cfg_weight=cfgw,
-        min_p=min_p,
-        top_p=top_p,
-        repetition_penalty=repetition_penalty,
-        steps=steps,
-        exaggeration=exaggeration
-    )
-    
-    if GLOBAL_MODEL is None:
-        raise RuntimeError("Model not initialized!")
+def _process_chunks(chunks: List[str], ref_wav: str, generation_config: GenerationConfig, seed_num: int) -> List[np.ndarray]:
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)